spec/features/deploy_spec.rb

```diff
@@ -148,11 +148,11 @@
     DeployService.new(strategy).start
   end
 
-  it 'notifies Slack prior to automatically merging release PR' do
+  it 'notifies one Slack prior to automatically merging release PR' do
     strategy.update!(arguments: strategy.arguments.merge(
       merge_after: 26.hours.to_i,
       merge_prior_warning: 75.minutes.to_i,
-      slack_webhook_url: 'https://hooks.slack.com/services/foo/bar/baz'
+      slack_webhook_url: ['https://hooks.slack.com/services/foo/bar/baz']
```

Comment:
This test could continue to use a string to test backward-compatibility.

Reply:
Hi @joeyAghion I have added compatibility for string, also added a test for ir